src/node_contextify.cc

```diff
@@ -929,6 +930,138 @@ class ContextifyScript : public BaseObject {
 };
 
 
+void ContextifyContext::GetWrappedFunction(
+    const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  Isolate* isolate = env->isolate();
+  Local<Context> context = env->context();
+
+  // Argument 1: source code
+  CHECK(args[0]->IsString());
+  Local<String> code = args[0].As<String>();
+
+  // Argument 2: filename
+  CHECK(args[1]->IsString());
+  Local<String> filename = args[1].As<String>();
+
+  // Argument 3: line offset
```

Comment:
i don't think we really need offsets

Reply:
I think we’d probably want this for parity with `New()`

Reply:
+1 to what @addaleax said. Added the functions for parity.